### PR TITLE
Add feature policy JS API.

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/feature/5190687460950016

### CODE REVIEW

File: api/Document.json [hunk 1]
```diff
@@ -5768,6 +5768,89 @@
...
+                {
+                  "type": "preference",
+                  "name": "#enable-experimental-productivity-features",
+                  "value_to_set": "Enabled"
+                }
+              ]
+            }
+          },
+          "status": {
+            "experimental": false,
```
Comment (Elchi3) [inline api/Document.json]: I guess this is experimental: true?

File: api/Document.json [hunk 2]
```diff
@@ -5768,6 +5768,89 @@
...
+            "standard_track": false,
+            "deprecated": true
```
Comment (Elchi3) [inline api/Document.json]: and not deprecated?

File: api/Document.json [hunk 3]
```diff
@@ -5768,6 +5768,89 @@
...
+                  "type": "preference",
+                  "name": "#enable-experimental-productivity-features",
+                  "value_to_set": "Enabled"
+                }
+              ]
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
```
Comment (Elchi3) [inline api/Document.json]: part of the draft spec or not?

File: api/Policy.json [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "allowsFeatures": {
```
Comment (Elchi3) [inline api/Policy.json]: allowedFeatures?

File: api/Policy.json [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Policy/allowsFeatures",
```
Comment (Elchi3) [inline api/Policy.json]: allowedFeatures?

**Comment by jpmedley** [comment]
Sorry about the errors.

**Pull request merged**